gaia-component.js

```diff
@@ -129,6 +61,10 @@ var base = {
       if (this.dirObserver) {
         document.removeEventListener('dirchanged', this.setInnerDirAttributes);
       }
+      if (document.l10n && this.onDOMRetranslated) {
+        document.l10n.ready.then(() => document.removeEventListener(
+          'DOMRetranslated', this.onDOMRetranslated));
+      }
```

Comment:
NIT: The rest of the code in this file has line breaks between blocks. Please follow the coding style in the file your working in.